templates/index.html

```diff
@@ -12,6 +12,9 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        Pick up Groceries
```

Comment:
This is a hardcoded list and not the list of saved todo items from `session_items.py`